File: spec/code_samples/Node_native/list@cleaned@{list_name}/delete.js
var http = require("http");

var options = {
  "method": "DELETE",
  "hostname": "example.com",
  "port": null,
  "path": "/api/v1/list/cleaned/%7Blist_name%7D?username=SOME_STRING_VALUE",
  "headers": {
    "content-type": "application/json",
    "authorization": "Bearer REPLACE_BEARER_TOKEN"
  }
};

var req = http.request(options, function (res) {
  var chunks = [];

  res.on("data", function (chunk) {
    chunks.push(chunk);
  });

  res.on("end", function () {
    var body = Buffer.concat(chunks);
    console.log(body.toString());
  });
});

req.end();

File: spec/code_samples/Python_python3/list@{list_id}/delete.py
import http.client

conn = http.client.HTTPConnection("example.com")

headers = {
    'content-type': "application/json",
    'authorization': "Bearer REPLACE_BEARER_TOKEN"
    }

conn.request("DELETE", "/api/v1/list/%7Blist_id%7D?username=SOME_STRING_VALUE", headers=headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))